editor/components/inserter-with-shortcuts/index.js

```diff
@@ -43,9 +43,7 @@ function InserterWithShortcuts( { items, isLocked, onToggle, onInsert } ) {
 					onClick={ () => onInsert( item ) }
 					label={ sprintf( __( 'Add %s' ), item.title ) }
 					icon={ (
-						<span className="editor-inserter-with-shortcuts__block-icon">
```

Comment:
I recall this span being added because of blocks with custom SVG icons but I think it's not necessary anymore because we don't mix text and icons in the same button like we used to do and this is taken care of at the IconButton level.